_security/access-control/document-level-security.md

```diff
@@ -185,3 +185,97 @@ plugins.security.dls.mode: filter-level
 Lucene-level DLS | `lucene-level` | This setting makes all DLS queries apply to the Lucene level. | Lucene-level DLS modifies Lucene queries and data structures directly. This is the most efficient mode but does not allow certain advanced constructs in DLS queries, including TLQs.
 Filter-level DLS | `filter-level` | This setting makes all DLS queries apply to the filter level. | In this mode, OpenSearch applies DLS by modifying queries that OpenSearch receives. This allows for term-level lookup queries in DLS queries, but you can only use the `get`, `search`, `mget`, and `msearch` operations to retrieve data from the protected index. Additionally, cross-cluster searches are limited with this mode.
 Adaptive | `adaptive-level` | The default setting that allows OpenSearch to automatically choose the mode. | DLS queries without TLQs are executed in Lucene-level mode, while DLS queries that contain TLQ are executed in filter- level mode.
+
+## DLS and multiple roles
+
+OpenSearch combines all DLS queries with the logical `OR` operator. However, when a role with DLS is combined with another security role that doesn't use DLS, the query results are filtered to display only documents matching the DLS from the first role. This filter rule also applies to roles which do not grant read documents.
+
+### When to enable `plugins.security.dfm_empty_overrides_all`
+
+When to enable the `plugins.security.dfm_empty_overrides_all` setting depends on whether you want to restrict users from accessing documents without DLS. 
+
+
+To ensure access is not restricted, you can set the following configuration in  `opensearch.yml`:
+
+```
+plugins.security.dfm_empty_overrides_all: true
+```
+{% include copy.html %}
+
+
+The following examples show what level of access roles with DLS enabled and without DLS enabled, depending on the interaction. These examples can help you decide when to enable the ``plugins.security.dfm_empty_overrides_all` setting.
```

Comment:
I'm not following "what level of access roles with..." Do we mean something like "show access role levels both with and without DLS enabled"?

Reply:
"Level of access" describes the permission level the roles are given. So maybe remove the word levels and say, "show access given both with and without DLS enabled?"